Source/Urho3D/Core/Profiler.h

```diff
@@ -37,8 +37,9 @@ void SetProfilerThreadName(const char* name);
 }
 
 #if URHO3D_PROFILING
-#   define URHO3D_PROFILE_C(name, color)          ZoneScopedNC(name, color)
-#   define URHO3D_PROFILE(name)                   ZoneScopedN(name)
```

Comment:
These lines were not modified but they are detected as changed. Probably because of inconsistent line endings. Could you please make sure there are no false-positive changes detected?